docs/source/dev/dockerfile/dockerfile.rst

```diff
@@ -0,0 +1,34 @@
+Dockerfile
+====================
+
```

Comment:
Could you add a short paragraph/description here on the multi-stage structure of the Dockerfile? 

Right now it looks a bit weird to show the visualization right away if the audience doesn't have too much context.

Reply:
Added a short description!